.github/workflows/apt-packages.yaml

```diff
@@ -4,6 +4,7 @@ name: APT packages
   schedule:
     # run daily 0:00 on main branch
     - cron: '0 0 * * *'
+  pull_request:
```

Comment:
I'm curious why we're adding this trigger and now testing packages on every PR. Could you elaborate on this?

Reply:
This is just for testing the workflow, i'll remove that line before merging.